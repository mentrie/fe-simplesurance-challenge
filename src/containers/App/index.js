import React, { Component } from 'react';
import Footer from '../../components/Footer';
import Questions from '../Questions';
import '../../static/styles/main.css';

class App extends Component {
  render() {
    return (
      <div>
        <main className="main">
         <Questions />
        </main>
        <Footer />
      </div>
    );
  }
}

export default App;
