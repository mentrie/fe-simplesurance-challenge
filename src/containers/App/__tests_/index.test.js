import React from 'react';
import App from '../index'

it('renders without crashing', () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});

it('It should contain Footer component', () => {
  const component = shallow(<App />);
  expect(component.find('Footer').length).toBe(1);
})
