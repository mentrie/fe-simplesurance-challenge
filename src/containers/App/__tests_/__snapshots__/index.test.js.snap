// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <main
    className="main"
  >
    <img
      alt="Simple surance logo"
      className="logo"
      src="simplesurance.png"
    />
    <Connect(Questions) />
  </main>
  <Footer />
</div>
`;
